### PR TITLE
blog: announce Tauri 1.7

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

**Review by amrbashir — APPROVED** [review]

File: src/content/docs/blog/tauri-1-7.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Upgrading
+
+Make sure to update both NPM and Cargo dependencies to the 1.7.0 release. You can update the dependencies with:
+
+<CommandTabs
+  npm="npm install @tauri-apps/cli@latest @tauri-apps/api@latest"
+  yarn="yarn upgrade @tauri-apps/cli @tauri-apps/api --latest"
+  pnpm="pnpm update @tauri-apps/cli @tauri-apps/api --latest"
+  cargo="cargo update"
```
Comment (simonhyll) [inline src/content/docs/blog/tauri-1-7.mdx:24]: Might as well tell people to also update the CLI
```suggestion
  cargo="cargo update && cargo install tauri-cli"
```

**Review by simonhyll — APPROVED** [review]
Small suggestion but I'd be ok with merging as-is as well

File: src/content/docs/blog/tauri-1-7.mdx (hunk 1 above)
Comment (amrbashir) [reply src/content/docs/blog/tauri-1-7.mdx:24]: this will confuse users who don't use the rust cli

**Comment by simonhyll** [comment]
It's fine if we ignore the formatting status check on this one, it's just the RPM article that's messed up.

**Pull request merged**